packages/property-grid/src/components/ElementList.tsx

```diff
@@ -177,9 +177,11 @@ export const ElementList = ({
       className={classnames("property-grid-react-element-list", rootClassName)}
     >
       <div className="property-grid-react-element-list-header">
+        {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
```

Comment:
could you add a role instead of disabling? https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-static-element-interactions.md